### PR TITLE
added progress bars to market bar

### PR DESCRIPTION
**Pull request by velinakennedy**

### CODE REVIEW

File: app/components/MarketBar.tsx [hunk 1]
```diff
@@ -1,53 +1,59 @@
...
+          }).format(data.data.total_market_cap[currency])}
+        </div>
+        <div className="flex items-center gap-2">
+          {new Intl.NumberFormat("en-US", {
+            maximumSignificantDigits: 3,
+            style: "currency",
+            currency: `${currency}`,
+            notation: "compact",
+            compactDisplay: "short",
+          }).format(data.data.total_volume[currency])}
```
Comment (yoloOnTheBattlefield) [inline app/components/MarketBar.tsx]: put this into a function

File: app/components/MarketBar.tsx [hunk 2]
```diff
@@ -1,53 +1,59 @@
...
+            currency: `${currency}`,
+            notation: "compact",
+            compactDisplay: "short",
+          }).format(data.data.total_market_cap[currency])}
+        </div>
+        <div className="flex items-center gap-2">
+          {new Intl.NumberFormat("en-US", {
+            maximumSignificantDigits: 3,
+            style: "currency",
+            currency: `${currency}`,
```
Comment (yoloOnTheBattlefield) [inline app/components/MarketBar.tsx]: currency, // you can use es6 shorthand here

File: app/components/MarketBar.tsx [hunk 3]
```diff
@@ -1,53 +1,59 @@
...
+          }).format(data.data.total_volume[currency])}
+          <ProgressBar percent={Math.floor((data.data.total_volume[currency]/data.data.total_market_cap[currency])*100)} fillColor="#ffffff" barColor="#8686a9" />
+        </div>
+        <div className="flex items-center gap-2">
+          {Math.floor(data.data.market_cap_percentage.btc)}%
+          <ProgressBar percent={Math.floor(data.data.market_cap_percentage.btc)} fillColor="#ea973d" barColor="#8686a9" />
+        </div>
+        <div className="flex items-center gap-2">
+          {Math.floor(data.data.market_cap_percentage.eth)}%
+          <ProgressBar percent={Math.floor(Math.floor(data.data.market_cap_percentage.eth))} fillColor="#849eff" barColor="#8686a9" />
```
Comment (yoloOnTheBattlefield) [inline app/components/MarketBar.tsx]: do you need double math.floor?

File: app/components/MarketBar.tsx [hunk 4]
```diff
@@ -1,53 +1,59 @@
...
+            style: "currency",
+            currency: `${currency}`,
+            notation: "compact",
+            compactDisplay: "short",
+          }).format(data.data.total_volume[currency])}
+          <ProgressBar percent={Math.floor((data.data.total_volume[currency]/data.data.total_market_cap[currency])*100)} fillColor="#ffffff" barColor="#8686a9" />
+        </div>
+        <div className="flex items-center gap-2">
+          {Math.floor(data.data.market_cap_percentage.btc)}%
+          <ProgressBar percent={Math.floor(data.data.market_cap_percentage.btc)} fillColor="#ea973d" barColor="#8686a9" />
```
Comment (yoloOnTheBattlefield) [inline app/components/MarketBar.tsx]: you can extract this logic into a component

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: app/components/MarketBar.tsx (hunk 4 above)
Comment (velinakennedy) [reply app/components/MarketBar.tsx]: I don't understand which part you want me to put into a component, all I can think of is to put that percentage logic into a separate function which I will definitely do but let me know if that's what you meant or if there was something else you wanted me to change

**Review by yoloOnTheBattlefield — APPROVED** [review]

**Pull request merged**